### PR TITLE
Docs move "visualizations" folder up one level in TOC

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
Moved the "visualizations" folder up one level, as discussed with @pkolyvas. This is part of user experience improvement by streamlining the TOC.

### CODE REVIEW

**Comment by torkelo** [comment]
this needs aliases for all the old paths (this cause an url change for all panels!)

**Review by matthewhelmke — APPROVED** [review]
LGTM

**Review by torkelo — DISMISSED** [review]
missing aliases required when moving files

**Comment by achatterjee-grafana** [comment]
> missing aliases required when moving files

I will add them. Thank you to both of you, for the review.

**Review by pkolyvas — APPROVED** [review]
So glad you ran with this @achatterjee-grafana - huge improvement for discoverability of key content.

File: docs/sources/administration/configuration.md [hunk 1]
```diff
@@ -1101,7 +1101,7 @@ Sets a global limit on number of alert rules that can be created. Default is -1
 
 ## [unified_alerting]
 
-For more information about the Unified Alerting feature in Grafana, refer to [Unified Alerting]({{< relref "../unified-alerting/_index.md" >}}}).
+For more information about the Grafana 8 alerts, refer to [Unified Alerting]({{< relref "../alerting/unified-alerting/_index.md" >}}).
```
Comment (Eve832) [inline docs/sources/administration/configuration.md:1104]: From a maintenance point of view - do we really want to brand the new alerting structure as Grafana 8 alerts? Seems like it'll just have to change for each release...this probably isn't the place to discuss this, but I thought I'd point it out, less I forget.

**Review by Eve832 — APPROVED** [review]

**Pull request merged**